### ISSUE TITLE
An exception occuring while implementing the combinedsignup&signin & invitationURL based signup&signin

### ISSUE THREAD

**Issue by rbagree** [azure-ad-b2c/samples#171]
I am trying to create a setup where a user can
1. signup & sign in directly from the combined signup&Signin page
2. Signup&signin from the invitation link.

Point one is working perfectly fine using the following files
1. [BaseFile](https://github.com/rbagree/B2CSignupSigninInvite/blob/main/TrustFrameworkBase.xml)
2. [ExtensionFile](https://github.com/rbagree/B2CSignupSigninInvite/blob/main/TrustFrameworkExtensions.xml)
3. [RPFile](https://github.com/rbagree/B2CSignupSigninInvite/blob/main/progressive_signup_signin.xml)

For point 2 I have created
1. [SignupinviteRPFile](https://github.com/rbagree/B2CSignupSigninInvite/blob/main/signup_invitation.xml)
Now, when I click on the invitation URL which is in the following format 

https://mytenant.b2clogin.com/mytenant.onmicrosoft.com/B2C_1A_signup_invitation/oauth2/v2.0/authorize?client_id={clientID}&nonce=ca00379642b94aa693a80b66783aa010&redirect_uri=https%3A%2F%2Fmytenant-dev.azurewebsites.net%2Fsignup%2Fuser-invite&scope=openid&response_type=id_token&id_token_hint={SignedJWTToken}

I do get the signup page with readonly emailID. But once I fill all the information and click "Create" it gives me 
![image](https://user-images.githubusercontent.com/17608412/106740911-081da500-6641-11eb-9a71-eefa7a4d3a0a.png)

Observations:
1. The account gets created in the local AD and I can see the user's entry
2. Following is the screenshot from the network logger 
![image](https://user-images.githubusercontent.com/17608412/106741187-6054a700-6641-11eb-9f9d-2be361fcab2e.png)
3. Following URL uses GET Method
https://mytenant.b2clogin.com/mytenant.onmicrosoft.com/B2C_1A_signup_invitation/api/SelfAsserted/confirmed?csrf_token=bThiL2hJNXZ4ZFBwSXZ3ZzRLd1lVUExQV2V1T3EzVkNBYUloaEpqWk5lYTBXczAvUW9oSjJMVXBEWWhrenZ1Ymc2SkJNL3N5N0UxNzZYNHBDVDdsaWc9PTsyMDIxLTAyLTAzVDExOjA2OjQ2LjU5NTgzMzVaO2tuVzlHdzdMTDZ1QzMyT1JmRGNZbGc9PTt7IlRhcmdldEVudGl0eSI6IkxvY2FsQWNjb3VudFNpZ25VcFdpdGhSZWFkT25seUVtYWlsIiwiT3JjaGVzdHJhdGlvblN0ZXAiOjN9&tx=StateProperties=eyJUSUQiOiIzYTlmMzVlNi01MWUxLTQwYjctOWVlOS1kOWM4MDgxZmY4ZDYifQ&p=B2C_1A_signup_invitation&diags=%7B%22pageViewId%22%3A%22e25ebe04-1601-460d-b3a8-1d958c8155b8%22%2C%22pageId%22%3A%22SelfAsserted%22%2C%22trace%22%3A%5B%7B%22ac%22%3A%22T005%22%2C%22acST%22%3A1612350407%2C%22acD%22%3A3%7D%2C%7B%22ac%22%3A%22T021%20-%20URL%3Ahttps%3A%2F%2Fmytenant.b2clogin.com%2Fstatic%2Ftenant%2Ftemplates%2FAzureBlue%2FselfAsserted.cshtml%3Fslice%3D001-000%26dc%3DPNQ%22%2C%22acST%22%3A1612350407%2C%22acD%22%3A50%7D%2C%7B%22ac%22%3A%22T019%22%2C%22acST%22%3A1612350407%2C%22acD%22%3A8%7D%2C%7B%22ac%22%3A%22T004%22%2C%22acST%22%3A1612350407%2C%22acD%22%3A2%7D%2C%7B%22ac%22%3A%22T003%22%2C%22acST%22%3A1612350407%2C%22acD%22%3A2%7D%2C%7B%22ac%22%3A%22T035%22%2C%22acST%22%3A1612350410%2C%22acD%22%3A0%7D%2C%7B%22ac%22%3A%22T030Online%22%2C%22acST%22%3A1612350410%2C%22acD%22%3A0%7D%2C%7B%22ac%22%3A%22T017T010%22%2C%22acST%22%3A1612350438%2C%22acD%22%3A1075%7D%2C%7B%22ac%22%3A%22T002%22%2C%22acST%22%3A1612350440%2C%22acD%22%3A0%7D%2C%7B%22ac%22%3A%22T017T010%22%2C%22acST%22%3A1612350438%2C%22acD%22%3A1077%7D%5D%7D

Gives following message:
![image](https://user-images.githubusercontent.com/17608412/106741514-cd683c80-6641-11eb-8969-30ba6de99825.png)

And the last call uses POST method
https://mytenant.b2clogin.com/mytenant.onmicrosoft.com/B2C_1A_signup_invitation/client/perftrace?tx=3a9f35e6-51e1-40b7-9ee9-d9c8081ff8d6&p=null

gives following error message
![image](https://user-images.githubusercontent.com/17608412/106741613-effa5580-6641-11eb-9327-2b60d0f17361.png)

Basically, after the signup from the invitation url I am not able to signin to my application. I am not sure if there is any conflict between the two RP files or If I am missing anything.

**Comment by JasSuri** [comment]
Paste your correlationid in text form please. Upload your policy files so we can see. 

There is no need to censor information (other than endpoints/urls you own inside the policy files).

Use app insights to get logs try narrow down the problem.
https://docs.microsoft.com/en-us/azure/active-directory-b2c/troubleshoot-with-application-insights

**Comment by rbagree** [comment]
The correlationID is: [3a9f35e6-51e1-40b7-9ee9-d9c8081ff8d6
[TrustFrameworkExtensions.txt](https://github.com/azure-ad-b2c/samples/files/5931763/TrustFrameworkExtensions.txt)
[TrustFrameworkBase.txt](https://github.com/azure-ad-b2c/samples/files/5931765/TrustFrameworkBase.txt)
[progressive_signup_signin.txt](https://github.com/azure-ad-b2c/samples/files/5931766/progressive_signup_signin.txt)
[signup_invitation.txt](https://github.com/azure-ad-b2c/samples/files/5931767/signup_invitation.txt)

I will try to look into the application insights

**Comment by JasSuri** [comment]
Answered on SO. 
https://stackoverflow.com/questions/66027343/azure-b2c-custom-policyan-exception-occuring-while-implementing-the-combinedsign/66070826#66070826
